src/DataTable/TablePagination.jsx

```diff
@@ -19,6 +19,7 @@ function TablePagination() {
       currentPage={pageIndex + 1}
       onPageSelect={(pageNum) => gotoPage(pageNum - 1)}
       pageCount={pageCount}
+      paginationLabel="table pagination"
```

Comment:
[consider] It may make sense to define the `paginationLabel` string as an i18n-compatible string so it's properly translated for non-English locales; the same could apply to the existing `paginationLabel` passed via `TablePaginationMinimal`.

For example, defining the `table pagination` string with `defineMessages` and using `intl.formatMessage` to return the translated message to the `paginationLabel` prop.